Source/WebCore/platform/graphics/cocoa/MediaPlayerPrivateWebM.h

```diff
@@ -328,6 +326,27 @@ class MediaPlayerPrivateWebM
     bool m_errored { false };
     bool m_processingInitializationSegment { false };
     Ref<WebAVSampleBufferListener> m_listener;
+
+    // Seek logic support
+    void seekToTarget(const SeekTarget&) final;
```

Comment:
I would put these methods above to not split the members in two sections.

Reply:
I like having the seek related in their own section better. There's two many virtual methods, it becomes hard to follow